client/src/shared-modules/Header.js

```diff
@@ -1,11 +1,37 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 function Header() {
-  return (
-    <div>
-      <h1>Header place holder</h1>
-    </div>
-  );
+	return (
+		<div >
+			<header>
+				<a href="#">Collective Foundation</a>
+
+				<nav>
+					<ul>
+						<li>
```

Comment:
List of items on the header should not be static. it should come from the backend so that when admin adds a new page, it should automatically appear on the header

Reply:
We don't have any back-end now, isn't it? Can you elaborate a bit more on how I should deal with it?